src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,63 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+
+        LocalDate localDateFrom = getDate(dateFrom);
+        LocalDate localDateTo = getDate(dateTo);
+
+        LocalDate [] consistencyLocalData = new LocalDate[data.length];
+        String [] consistencyEmployee = new String[data.length];
+        int [] consistencyWorkingHourDuringTheParticularDay = new int [data.length];
+        int [] consistencyIncomePerHour = new int[data.length];
+
+        int [] sum = new int[names.length];
+
+        StringBuilder stringBuilder = new StringBuilder();
+        stringBuilder.append("Report for period ").append(dateFrom).append(" - ").append(dateTo);
+
+        //записуєм data в різні масиви, з конкретною інформаціїєю
+        for (int i = 0; i < data.length; i++) {
+            String [] dataSplit = data[i].split(" ");
+
+            consistencyLocalData[i] = getDate(dataSplit[0]);
```

Comment:
magic numbers, all indexes in dataSplit array mean somethig, so please create constants